cmd/completion.go

```diff
@@ -57,6 +57,12 @@ var completionExample = `
 	# Set the dapr completion code for zsh[1] to autoload on startup
   	dapr completion zsh > "${fpath[1]}/_dapr"
 
+	# Installing fish completion on Linux
```

Comment:
Can it be also used on macOS? Also is any `fish-completion` package required on either Linux or macOS?

Reply:
Sorry I am not familiar with macOS or windows environment.

 `fish-completion` is not required, it's a built in feature.